Signal/src/network/PushManager.m

```diff
@@ -85,14 +85,16 @@ - (instancetype)initWithNetworkManager:(TSNetworkManager *)networkManager
 
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(handleMessageRead:)
-                                                 name:OWSReadReceiptsProcessorMarkedMessageAsReadNotification
+                                                 name:kMessageMarkedAsReadNotification
                                                object:nil];
 
     return self;
 }
 
 - (void)handleMessageRead:(NSNotification *)notification
 {
+    OWSAssert([NSThread isMainThread]);
```

Comment:
I try to always post notifications on the main thread.  This one doesn't technically need to be on the main thread, but being consistent is worth the waste?